e2e/utils.go

```diff
@@ -668,7 +668,7 @@ func validatePVCClone(totalCount int, sourcePvcPath, sourceAppPath, clonePvcPath
 }
 
 // nolint:gocyclo,gocognit,nestif // reduce complexity
-func validatePVCSnapshot(totalCount int, pvcPath, appPath, snapshotPath, pvcClonePath, appClonePath, kms string, validateEncryption bool, f *framework.Framework) {
+func validatePVCSnapshot(totalCount int, pvcPath, appPath, snapshotPath, pvcClonePath, appClonePath, kms string, f *framework.Framework) {
```

Comment:
You could simplify the changes to the remaining of the function if you add
```go
validateEncryption := (kms != "")
```

Reply:
isn't it the same as passing the true or false to this variable? As you suggested in https://github.com/ceph/ceph-csi/pull/2095, we can avoid using validateEncryption, if our work is done using kms?

Reply:
There is no need to pass `validateEncryption` as parameter to the function, but using it inside the function by 'calculating' is fine (and might be more readable).

Reply:
Yeah that's true. I will do that.